src/buttons/src/TextDropdownButton.js

```diff
@@ -6,15 +6,7 @@ import { CaretDownIcon } from '../../icons'
 import { Text } from '../../typography'
 import { Spinner } from '../../spinner'
 import { useTheme } from '../../theme'
-
-const styles = {
-  position: 'relative',
-  fontFamily: 'ui',
-  fontWeight: 500,
-  display: 'inline-flex',
-  alignItems: 'center',
-  flexWrap: 'nowrap'
-}
+import { styles } from './Button'
```

Comment:
What about just importing the `<Button />` component, and overriding some of the box props that are passed in? Don't think the glamor styles are too different

Reply:
tbh this should be a variant... "unstyled"